pkg/bundle/manager.go

```diff
@@ -19,7 +19,7 @@ import (
 
 type Manager interface {
 	// Update the bundle returns true if there are changes
```

Comment:
The pattern where we pass in the state to `Update` is opposite of the pattern I have been trying to follow here. I've been trying to get around to where `isActive` is not passed in here. It has been difficult getting rid of that and this adds another state we are passing.

Reply:
Fixed in the latest commit